packages/svelte2tsx/svelte-jsx.d.ts

```diff
@@ -1,5 +1,286 @@
 /// <reference lib="dom" />
-/* eslint @typescript-eslint/no-unused-vars: off */
+
+declare namespace svelte {
```

Comment:
This namespace definition is mostly a copy-paste from the `svelte.JSX` namespace. Reason: I want to get rid of the `JSX` fragment in there, it confuses people ("wait why is this JSX?"). The old version exists for backwards compatibility, for example we a section in our docs where we tell people to enhance the `svelte.JSX` namespace; we would break everyone who did this, so we should keep the old version 1-2 years.